Point to pkg-registration channel for manual merge of repo transfer (#106692)
JuliaRegistries/General: README.md

 - Use the [GitHub transfer option][github-transfer] in the settings.
 - Make a pull request to [this repository](https://github.com/JuliaRegistries/General/pulls) in which you edit the repo URL in the package's Package.toml file (e.g [E/Example/Package.toml](https://github.com/JuliaRegistries/General/blob/master/E/Example/Package.toml#L3)).

>  - Ask for a manual merge in the `#pkg-registration` Slack channel if the PR is not attended to or if you have any questions.

Reviewer: ```suggestion
 - If the PR is not attended to, or if you have any questions, you can ask for help in the `#pkg-registration` Slack channel.
```
